Library/Formula/coreutils.rb

```diff
@@ -29,6 +29,7 @@ class Coreutils < Formula
     depends_on "texinfo" => :build
     depends_on "xz" => :build
     depends_on "wget" => :build
+    depends_on "gperf" => :build
```

Comment:
But this part is inside of a `head` section. Shouldn't it be just `depends_on "homebrew/dupes/gperf" => :build && !OS.mac?`